test/integration/etcd_storage_path_test.go

```diff
@@ -1250,17 +1252,14 @@ type etcd3Getter struct {
 }
 
 func (e *etcd3Getter) getFromEtcd(path string) (*metaObject, error) {
-	response, err := e.kv.Get(context.Background(), path, etcdv3.WithSerializable())
+	response, err := e.kv.Get(context.Background(), "/"+path) // TODO(enj/post-1.6.1-rebase): change all paths to have a leading slash
```

Comment:
I knew the rebase is a terrible time to mess with the paths so this hack is needed to make etcd3 pass.